en/render-pipeline/builtin-pipeline.md

```diff
@@ -0,0 +1,25 @@
+# Builtin Rendering Pipeline
+
+As of version 3.1, the builtin rendering pipeline includes builtin-forward and builtin-deferred. The rendering pipeline can be switched via **Project->Project Settings->Project Data->RenderPipeline**, and restart the editor after the switch to take effect.
```

Comment:
```suggestion
The builtin rendering pipeline includes builtin-forward and builtin-deferred. The rendering pipeline can be switched via **Project->Project Settings->Project Data->RenderPipeline**, and restart the editor after the switch to take effect.
```

Reply:
We don't need to say "As of v3.1" since this is the 3.1 documentation.

Reply:
> We don't need to say "As of v3.1" since this is the 3.1 documentation.

Updated